src/implementation/Client/HTTPClient/HTTPClient.ts

```diff
@@ -153,6 +153,13 @@ export default class HTTPClient implements IClient {
       params.headers = {};
     }
 
+    if (this.options.daprApiToken) {
```

Comment:
Please rework this to: 

```javascript
if (this.options.daprApiToken) {
    params.headers["dapr-api-token"] = this.options.daprApiToken;
}
```

I might even want to see: 

```javascript
params.headers["dapr-api-token"] = this.options.daprApiToken || undefined; 
```

but then I would like @shubham1172 his opinion

Reply:
Agreed, first one looks cleaner (without `undefined`).

Reply:
I agree too.